bundles/org.openhab.ui.dashboard/src/main/java/org/openhab/ui/dashboard/internal/DashboardServlet.java

```diff
@@ -75,7 +75,6 @@ private void serveDashboard(HttpServletRequest req, HttpServletResponse resp) th
         for (DashboardTile tile : tiles) {
             String entry = entryTemplate.replace("<!--name-->", tile.getName());
             entry = entry.replace("<!--url-->", tile.getUrl());
-            entry = entry.replace("<!--overlay-->", tile.getOverlay());
```

Comment:
So why do you remove this line? This means that a tile overlay won't have any effect anymore.
Imho, we should simply allow getOverlay() to return null and handle that accordingly.